web/html/src/manager/content-management/list-filters/list-filters.renderer.tsx

```diff
@@ -22,6 +25,10 @@ export const renderer = (id, { filters, projectLabel, openFilterId, flashMessage
           filters={filtersJson}
           openFilterId={openFilterId}
           projectLabel={projectLabel}
+          openTemplate={openTemplate}
+          systemId={systemId}
+          systemName={systemName}
+          kernelName={kernelName}
```

Comment:
One thing that bothers me a bit is this horizontal expansion of template-specific parameters. It looks like it will keep growing as we add more templates. Can we maybe wrap these in an object and call it something like `templateValues`?

Reply:
Fetching the URL params in the component itself should fix this issue, or is there anything else you'd like to address here @cbbayburt?